.github/workflows/version.yml

```
@@ -297,3 +309,99 @@ jobs:
+      - name: Publish or verify existing stable npm package
+        working-directory: .stable-source
+        env:
+          EXPECTED_VERSION: ${{ inputs.expected_version }}
+          NPM_RECOVERY_TOKEN: ${{ secrets.NPM_TOKEN }}
+          HUSKY: "0"
+        run: |
+          set -euo pipefail
+          "${GITHUB_WORKSPACE}/scripts/release/reconcile-npm-stable.sh" \
+            --expected "${EXPECTED_VERSION}" \
+            --package-dir "${GITHUB_WORKSPACE}/.stable-source/packages/cli"
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)